src/components/blog/SingleBlogCard.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Col, Card, Button, ListGroupItem, ListGroup } from "react-bootstrap";
+
+export default function SingleBlogCard(props) {
+  const renderCard = (
+    <Card>
+      {props.blogImage && (
+        <Card.Img variant="top" src={props.blogImage.source_url} />
+      )}
+      <Card.Body>
+        <Card.Title>{props.blog.title.rendered}</Card.Title>
+      </Card.Body>
+      <Card.Body>
+        <Card.Text
+          dangerouslySetInnerHTML={{
+            __html: props.blog.excerpt.rendered.substring(0, 100) + " ...",
```

Comment:
i don't think you want to take the excerpt like so, you want to take the whole excerpt. if the excerpt is too long then the author should modify it in the WP UI (they can customize their exverpt)